src/App.tsx

```diff
@@ -306,8 +176,6 @@ const App = (props: KorgProgramDump) => {
       <button onClick={() => selectPatch(fake3OSC)}>Fake30OSC</button>
       <button onClick={() => selectPatch(TeeVeeSaw)}>TeeVeeSaw</button>
       <button onClick={() => connectMidi()}>ConnectMidi</button>
-      <button onClick={() => listen()}>Listen</button>
```

Comment:
This listen is now happening in index.ts, as the already connected and listening midi connection is passed through as a dependency.